src/melee.cpp

```diff
@@ -1965,14 +1973,12 @@ void player::disarm( npc &target )
     their_roll += dice( 3, target.get_per() );
     their_roll += dice( 3, target.get_skill_level( skill_melee ) );
 
-    // make the target angry as they confess our unfriendly move
-    target.make_angry();
-
     // roll your melee and target's dodge skills to check if grab/smash attack succeeds
     int hitspread = target.deal_melee_attack( this, hit_roll() );
     if( hitspread < 0 ) {
         // this will not do damage, but trigger all miss effects and on_dodge on target
         melee_attack( target, false, no_technique_id, hitspread );
+        aggress_npc( target );
```

Comment:
For ordering of messages in the log. If aggressing first, we see the target get angry followed by the miss. This allows for what I hoped was a more reasonable flow where one would see the attempt made, followed by noting the NPC getting angry.